app/src/components/Map.jsx

```diff
@@ -136,13 +136,18 @@ class Map extends Component {
       return;
     }
     this.state.trackLayer.recalculatePosition();
-
     this.state.trackLayer.drawTile(
       workProps.vesselTrack.seriesGroupData,
       workProps.vesselTrack.selectedSeries,
       workProps.filters,
-      workProps.map.vesselTrackDisplayMode
+      workProps.map.vesselTrackDisplayMode,
+      // I don't understand why this array has way more points than it should
+      // workProps.vesselTrack.seriesGroupData.weight,
+      1,
+      this.state.zoom
     );
+    // this shouldn't be called when no apparent track shows up, which I don't know how to test
+    this.state.overlay.dim();
```

Comment:
that's something we need to add: when a user clicks on a non-vessel pixel, hide the track + clear this state. might also be related to the issue where we need to filter the clickable point by innerbounds, instead of the outter bounds